Mobile Buy SDK/Mobile Buy SDK Tests/test_shop_data.json

```diff
@@ -5,8 +5,8 @@
 	"channel_id": "",
 	"app_id": "",
 	"merchant_id": "",
-	"customer_email": "",
-	"customer_password": "",
+	"customer_email": "asd@asd.com",
+	"customer_password": "asdasd",
```

Comment:
This is necessary for local test to run properly. Perhaps in the future, we can investigate rewriting tests to run without this requirement.